### PR TITLE
Open Rails homepage in new tab

### PR DESCRIPTION
**Pull request by excid3**
With the new boot screen that @dhh just added, it would be very helpful to include links to the Rails Guides, API docs, etc.

By including the Rails version in the Guides and API links, we can take the user directly to the correct version of the Guides and API docs. Also, `target="blank"` helps keep their browser tab open so they don't lose it.

These links are also helpful to new users for quick access to documentation on their newly created apps.

Screenshot of navigation links
Screen Shot 2021-12-08 at 8 46 49 AM

### CODE REVIEW

**Comment by dhh** [comment]
I'm not sure this really adds much. The Rails logo itself is a link to a page that has all those things on it. I'd rather keep this boot screen clean. I could be persuaded of opening in a new window, though.

**Comment by kylekeesling** [comment]
I understand the goal of keeping it clean, but another plus of adding (at least some of) these links would be the ability to link to a specific version of the docs/guides directly, making it easier for someone new.

**Comment by onerinas** [comment]
> I understand the goal of keeping it clean, but another plus of adding (at least some of) these links would be the ability to link to a specific version of the docs/guides directly, making it easier for someone new.

When someone installs it, I would assume its on latest version? 

I like the simpler version we have right now. If we were to make it useful linking docs to installed version, may be it can be in the link logo has?

**Comment by excid3** [comment]
Another use case would be for linking to Rails Conductor (when that gets finished) and Mailer Previews in development. There's currently no way for users to easily discover those features easily and this would be a good place for it.

File: railties/lib/rails/templates/rails/welcome/index.html.erb [hunk 1]
```diff
@@ -70,12 +70,31 @@
...
+    }
+
+    header a {
+      margin: 0.5rem;
+    }
   </style>
 </head>
 
 <body>
+  <header>
```
Comment (MatheusRich) [inline railties/lib/rails/templates/rails/welcome/index.html.erb:88]: Should we use `nav` here as well?

Comment (excid3) [reply railties/lib/rails/templates/rails/welcome/index.html.erb:88]: It would conflict with the CSS for `nav` that's used to display the Rails logo.

**Comment by dhh** [comment]
Yeah, ultimately I'd love to get all the way to conductor. But it's not going to be in 7.0. So for now, I'm not digging the extra navigation added here. I'll take the target=_new though.

**Comment by excid3** [comment]
Sounds good, I'll update this PR and we can revisit later. 👍

**Comment by excid3** [comment]
PR is updated @dhh 👍

**Comment by lafeber** [comment]
A little late to the party, but FYI: https://stackoverflow.com/questions/4964130/target-blank-vs-target-new

**Comment by dhh** [comment]
@lafeber Fixed. Thanks.

**Pull request merged**